### DISCUSSION TITLE
Welcome to signalk-cloud Discussions!

### DISCUSSION THREAD

**Discussion by sbender9** [sbender9/signalk-cloud#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jouansteph** [comment]
hi everyone,
i have finally terminate my RaspberryPi setup, tested, and configured the connection with the CloudServer, but since this morning, this server is down,.. 
what should i get to create and host my own signalK Cloud server? 
i dispose currently of a physical server linux, on LAMP, and dns name..

thanks about this hint :)

**Reply by sbender9** [reply]
The issue was on the server. Should be back up now.

You can run your own server, just run the sk server on the cloud somewhere, create a token with `signalk-generate-token` and point the cloud plugin there.

**Reply by jouansteph** [reply]
thanks,
but about this server, what's are the requirement's (setup i mean) to have?

**Reply by sbender9** [reply]
You just need to be able to install and run the server

**Reply by sbender9** [reply]
The existing cloud server runs on a small Amazon Ubuntu EC2 instance

**Reply by jouansteph** [reply]
nice,.. any link to do this?

**Reply by sbender9** [reply]
I don't think anyone has written this up. It really is as simple as installing the server (like you would on a rpi): https://github.com/SignalK/signalk-server

**Reply by hwater** [reply]
after the latest signalk-server update from 1.41.0 to 1.41.2 this plugin refuses to start:

signalk-cloud failed to start: Package subpath './v4' is not defined by "exports" in /home/pi/.signalk/node_modules/uuid/package.json

**Reply by albuchan** [reply]
Getting the same error as above since making a new build on 1.44 and restoring SignalK + plugins from backup. Reinstalling plug-in does not fix. Appreciate any hints. Thx

**Reply by pancirm** [reply]
Hello. The same error... Has everybody solution?

**Reply by dufaza** [reply]
Hello. 
Using Victron Energy Venus OS Large v2.93, with SignalK and trying to install Signalk-cloud ... but still the same error.
Error Msg = Feb 22 14:20:42 signalk-cloud failed to start: Package subpath './v4' is not defined by "exports" in /data/conf/signalk/node_modules/uuid/package.json 
So cannot use Wilhelmsk and so on ...
Any help, thanks

**Reply by seamaster101** [reply]
I’m also getting “Failed to start: Package subpath './v4” error and can not run the cloud plugin. Please someone reply with solution? I have my boat left in a marina in Mexico but I have full access to the boat PC and the RPI console remotely. Please help me fix the issue so I can access the boat data remotely while I’m away for the next 3-4 months.

**Comment by seamaster101** [comment]
@sbender9 please help us. I’m trying to use WilhelmSK, but running into the above difficulty when trying to access remotely

**Comment by alnavasa** [comment]
love the idea, but any plans on making data private and not public?
